openfe/tests/protocols/test_openmm_afe_solvation_protocol.py

```diff
@@ -33,17 +33,28 @@ def test_create_default_settings():
 
 
 @pytest.mark.parametrize('val', [
-    {'elec': 0, 'vdw': 5},
-    {'elec': -2, 'vdw': 5},
-    {'elec': 5, 'vdw': -2},
-    {'elec': 5, 'vdw': 0},
+    {'elec': [0.0, -1], 'vdw': [0.0, 1.0], 'restraints': [0.0, 1.0]},
+    {'elec': [0.0, 1.5], 'vdw': [0.0, 1.5], 'restraints': [-0.1, 1.0]}
 ])
 def test_incorrect_window_settings(val, default_settings):
-    errmsg = "lambda steps must be positive"
-    alchem_settings = default_settings.alchemical_settings
+    errmsg = "Lambda windows must be between 0 and 1."
+    lambda_settings = default_settings.lambda_settings
     with pytest.raises(ValueError, match=errmsg):
-        alchem_settings.lambda_elec_windows = val['elec']
-        alchem_settings.lambda_vdw_windows = val['vdw']
+        lambda_settings.lambda_elec = val['elec']
+        lambda_settings.lambda_vdw = val['vdw']
+        lambda_settings.lambda_restraints = val['restraints']
+
+@pytest.mark.parametrize('val', [
+    {'elec': [0.0, 0.1, 1.0], 'vdw': [0.0, 1.0], 'restraints': [0.0, 1.0]},
+])
+def test_incorrect_number_windows(val, default_settings):
+    errmsg = "Components elec, vdw and restraints must have equal amount " \
+             "of lambda windows"
```

Comment:
```suggestion
    errmsg = ("Components elec, vdw and restraints must have equal amount "
              "of lambda windows")
```

As per our discussion at today's meeting - not a big deal just a nice PEP8 flair thing